lib/rsgem/gem.rb

```diff
@@ -39,12 +41,20 @@ def clean_gemfile
       Tasks::CleanGemfile.new(context: context).clean
     end
 
+    def clean_gemspec
+      Tasks::CleanGemspec.new(context: context).clean
+    end
```

Comment:
I think you are missing tests for this

Reply:
It's hard to test and I don't think it will add value. If this gemspec file is malformed or with FIXME or TODOs, the rubocop task will raise an error. 

Given we're already testing rubocop and reek passes after creating the gem, I think we are good. It will fail if the gemspec is malformed.